model/container/deprecated/registry-deprecated.yaml

```diff
@@ -29,3 +29,17 @@ groups:
         stability: experimental
         deprecated: 'Replaced by `cpu.mode`'
         examples: [ "user", "kernel" ]
+
+  - id: registry.container.metrics.deprecated
+    type: attribute_group
+    display_name: Deprecated Container Metrics
+    brief: >
+      "Describes deprecated container metrics attributes."
+    stability: experimental
+    attributes:
+      - id: system.device
+        type: string
+        stability: experimental
+        brief: "Deprecated, use `network.interface.name` instead."
+        deprecated: "Replaced by `network.interface.name`."
+        examples: [ "(identifier)" ]
```

Comment:
```suggestion
```

since you're declaring it deprecated in system/deprecated/registry-deprecated.yaml, no need to add anything here.

the file structure is only important for humans, tooling does not care about file names.

Reply:
Wait, from https://github.com/open-telemetry/semantic-conventions/pull/1492#issuecomment-2425987628 I jus realized that the attribute is still used. Let's not deprecate it at all then, just use a new one on the metrics it's applicable to.